infra/modules/container-app.bicep

```diff
@@ -69,7 +70,7 @@ module containerApp 'br/public:avm/res/app/container-app:0.8.0' = {
     }
     containers: [
       {
-        image: fetchLatestImage.outputs.?containers.?value[?0].?image ?? 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
+        image: fetchLatestImage.?outputs.?containers.?value[?0].?image ?? existingImage ?? 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
```

Comment:
if this value is in the .env file, should you always use that value and skip the lookup of the image that is deployed to Container Apps?

Reply:
as far as I know AZD does provide it automatically. In some examples, I've seen container app imported (`existing` keyword) to read the value of the image too.